torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -472,6 +507,8 @@ void Reducer::initialize_buckets(
           .bucket_index = bucket_index,
           .intra_bucket_index = intra_bucket_index++,
       };
+
+      bucket.variable_indices.push_back(variable_index);
```

Comment:
`bucket.variable_indice`'s content is the same as `bucket_indices[bucket_index]`? If so, is it OK to move the latter into the former?

Reply:
Good point. Done as cc28ee6.